requirements.txt

```diff
@@ -13,6 +13,15 @@
 # often breaking changes across major versions.  This is the reason
 # for the upper bound.
 ansible>=8,<10
+# TODO: Remove this pin when possible.  See
+# cisagov/skeleton-packer##312 for more details.
```

Comment:
As @dav3r mentioned in https://github.com/cisagov/skeleton-ansible-role/pull/179#discussion_r1518905639.

```suggestion
# cisagov/skeleton-packer#312 for more details.
```